plenum/server/message_req_processor.py

```
@@ -123,7 +123,7 @@ def _validate_requested_cons_proof(self, **kwargs):
                     # the input is expected as a dict (serialization with ujson==1.33)
                     return ConsistencyProof(**kwargs['cons_proof'])
                 except TypeError as ex:
-                    logger.warning(
```

Let's not hide this exception. I think warning or error should be fine